openfl-tutorials/experimental/Workflow_Interface_101_MNIST.ipynb

```diff
@@ -390,6 +391,15 @@
    "cell_type": "markdown",
    "id": "86b3dd2e",
    "metadata": {},
+   "source": [
+    "In case $USERNAME is empty, metaflow will throw MetaflowUnknownUser Error.\n",
+    "To resolve it, set the $USERNAME environment variable. For example, you can use the command export USERNAME='your-name' to set the value of the $USERNAME variable to your desired name."
```

Comment:
You might want to use `\$USERNAME` (escape the $) as the UI shows a mangled block between the two dollar symbols. 
<img width="933" alt="Screenshot 2024-10-24 at 4 47 09 PM" src="https://github.com/user-attachments/assets/b2b58b08-b2d1-4b35-9dab-82ff44d30d3f">

Reply:
removed it as this is not required.